apply/applydriver/local.go

```diff
@@ -213,11 +231,55 @@ func (c *Applier) upgradeCluster(mj, nj []string) error {
 		return err
 	}
 
-	logger.Info("Succeeded in upgrading current cluster from version(%s) to version(%s)", info.GitVersion, clusterMetadata.Version)
+	logger.Info("Succeeded in upgrading current cluster from version(%s) to version(%s)", c.CurrentClusterInfo.GitVersion, upgradeImgMeta.Version)
+	return utils.SaveClusterInfoToFile(c.ClusterDesired, c.ClusterDesired.Name)
+}
+
+func (c *Applier) CheckDiffImage(imageName string) error {
+	if c.ClusterDesired.Spec.Image == imageName {
```

Comment:
Check current cluster version == ClusterImage kubernetes version

Reply:
> Check current cluster version == ClusterImage kubernetes version

This is already implemented in line 219.